test/Table_Tests/src/In_Memory/Column_Spec.enso

```diff
@@ -432,6 +432,12 @@ add_specs suite_builder =
                 c.truncate.to_vector . should_equal c.to_vector
                 c.truncate.length . should_equal c.length
 
+        group_builder.specify "Use Integer Column as index" <|
+            file1 = Table.new [["A", [1]]]
+            column1 = file1.at 'A'
+            table1 = file1.take (..First column1)
```

Comment:
- 62bed183b004ab3e6f1343b4abb560736ad9b040 explains the purpose of this PR in a nutshell
- if you want to use `Column` as an Integer then replace
- `Column & In_Memory_Column & Numeric_Column` by
- `Column & In_Memory_Column & Numeric_Column & Any`